com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/StepsUtil.java

```diff
@@ -1177,4 +1129,37 @@ public static String getString(VariableScope scope, String name, String defaultV
         return value == null ? null : factory.apply(value);
     }
 
+    public static void setAsJsonBinary(VariableScope scope, String name, Object value) {

```

Comment:
Why is the variableScope used and not the DelegateExecution?

Reply:
Two reasons:
# Getters/Setters do not need the additional functionality `DelegateExecution` brings. Therefore, using it is kinda like using `ArrayList` instead of `List` everywhere.
# `VariableScope` is much more readable as a name when seen in a getter/setter.